SMPyBandits/Arms/RestedRottingArm.py

```diff
@@ -0,0 +1,69 @@
+"""
+author: Julien Seznec
+Rested rotting arm, i.e. arms with mean value which decay at each pull
+"""
```

Comment:
That's a great start. Could you:
- put the documentation first, with maybe a little bit more details?
- then add a reference to a paper with:

```rst
- Reference: blabla
- Author: Julien Seznec
```

Reply:
Isn't it weird to put a reference for an arm ? I don't think any paper can claim the "ownership" of an arm. Relevant Rotting bandits paper are cited in the algorithm code and in the experiment file (see other PRs)

 Here, "author" is the author of the code.